_layouts/mainhomepage.html

```diff
@@ -22,7 +22,7 @@
       "isAccessibleForFree": true,
       "alternateName": "llcon",
       "thumbnailUrl": "https://jamulus.io/assets/img/jamulus-icon-2020.svg",
-      "downloadUrl": "https://sourceforge.net/projects/llcon/files/latest/download",
+      "downloadUrl": "https://github.com/jamulussoftware/jamulus/releases/latest",
```

Comment:
Are you sure about that? Shouldn't we wait until a download page is created: https://github.com/jamulussoftware/jamuluswebsite/issues/322

Reply:
Hrm, I thought we'd move all download links to Github? I agree that a real download page would be best, but as it isn't ready yet, I'd suggest replacing the already technical, confusing and ad-loaded SourceForge link by Github which is still technical but hopefully less confusing and without ads.

Reply:
Yes. Maybe most users are probably not clicking on this button anyway.